### ISSUE TITLE
English (United States) is listed twice 

### ISSUE THREAD

**Issue by mixxxbot** [mixxxdj/mixxx#9638]
Reported by: **[daschuer](https://github.com/daschuer)**
Date: 2019-04-20T13:13:56Z
Status: Confirmed
Importance: Medium
Launchpad Issue: [lp1825644](https://bugs.launchpad.net/bugs/1825644)
Tags: usability

---

English (United States) is listed twice in Language Preferences Combobox. 
Many other languages are effected as well. 

I have noticed that with current master 2.3.1-alpha-pre.

**Comment by foss-** [comment]
Mixxx 2.5-alpha-84-gde4e2cca61 (main)
Qt 6.5.2

No longer reproducible. @daschuer  can you confirm?

![1](https://github.com/mixxxdj/mixxx/assets/5708172/dcfafe7a-428f-4d9c-8a4b-9882648c1ea7)

**Comment by foss-** [comment]
Isn't https://github.com/mixxxdj/mixxx/issues/10060 a duplicate of this issue here?

**Comment by ronso0** [comment]
I think `en_US` is the translation source and it's listed only once.



> Isn't #10060 a duplicate of this issue here?

Nope, #10060 is about identical (base) languages for different countries where only minor differences are to be expected which lead to translation files with very low coverage.
